distributed/client.py

```diff
@@ -1377,8 +1382,6 @@ async def _close(self, fast=False):
 
         self.status = "closed"
 
-    _shutdown = _close
-
```

Comment:
Just noting to future readers that there's a `_shutdown` method defined a few lines below